src/Symfony/Component/Form/Tests/Extension/Csrf/CsrfProvider/LegacyDefaultCsrfProviderTest.php

```diff
@@ -25,8 +25,10 @@ class LegacyDefaultCsrfProviderTest extends TestCase
 
     public static function setUpBeforeClass()
     {
-        ini_set('session.save_handler', 'files');
-        ini_set('session.save_path', sys_get_temp_dir());
+        if (PHP_VERSION_ID < 70200) {
```

Comment:
`PHP_VERSION_ID` should be fully-qualified instead of relying on constant fallback, to allow resolving the version check at compilation time.

Reply:
What do you mean ? (`PHP_VERSION` maybe ?)

Reply:
whoops, understood